src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -81,6 +81,7 @@ import ShareDialog from '../ShareDialog/ShareDialog';
 import { sharedThunks } from '../../../store/slices/sharedLinks';
 import { fetchSortedFolderContentThunk } from 'app/store/slices/storage/storage.thunks/fetchSortedFolderContentThunk';
 import envService from '../../../core/services/env.service';
+import BannerWrapper from 'app/banners/BannerWrapper';
```

Comment:
Try to avoid absolute imports, instead do: '../../../something' as when we want to perform some configurations, these kind of imports make it impossible

Reply:
Done